lib/ModelMixins/TableMixin.ts

```diff
@@ -717,7 +717,7 @@ function TableMixin<T extends Constructor<Model<TableTraits>>>(Base: T) {
 
                 return this.featureInfoFromFeature(
                   regionType,
-                  d,
+                  Object.assign({}, feature.properties, d),
```

Comment:
```suggestion
                  Object.assign(
                    {},
                    d,
                    // Only add feature.properties which don't exist in $d
                    Object.entries(feature.properties).reduce<any>(
                      (props, [key, value]) => {
                        if (d![key] === undefined) {
                          props[key] = value;
                        }
                        return props;
                      },
                      {}
                    )
                  ),
```

Is it worth reversing the order of properties so column values are listed above region feature properties?

Reply:
Good catch. I raise your suggestion with:
```suggestion
                  Object.assign(d, Object.assign({}, feature.properties, d)),
```

I know this modifies d, but we were modifying it previously in `featureInfoFunc`. Or:
```suggestion
                  Object.assign({}, d, Object.assign({}, feature.properties, d)),
```
Or we could use what you've written if you think that conveys intent better.


(the same as above but with a quick comment, but maybe not a good comment)
```suggestion
                  // Preserve values from d and insert feature properties after entries from d
                  Object.assign(d, Object.assign({}, feature.properties, d)),
```

Not modifying d:
```suggestion
                  // Preserve values from d and insert feature properties after entries from d
                  Object.assign({}, d, Object.assign({}, feature.properties, d)),
```

Reply:
Yep the last one looks good to me!
But is it equivalent to `Object.assign({}, d, feature.properties, d)`?

![image](https://user-images.githubusercontent.com/6187649/106723699-7d966f00-665b-11eb-88a0-81ff5c9f16df.png)

Reply:
> Properties in the target object are overwritten by properties in the sources if they have the same key. Later sources' properties overwrite earlier ones.

Reply:
Yep. That's equivalent

Reply:
```suggestion
                  // Preserve values from d and insert feature properties after entries from d
                  Object.assign({}, d, feature.properties, d),
```